packages/esm-appointments-app/src/index.ts

```diff
@@ -43,6 +51,12 @@ export function startupApp() {
   defineExtensionConfigSchema('cancelled-appointments-panel', cancelledAppointmentsPanelConfigSchema);
   defineExtensionConfigSchema('early-appointments-panel', earlyAppointmentsPanelConfigSchema);
 
+  registerWorkspace({
```

Comment:
I was able to set up the appointment form in the patient chart context using the register workspace function how that everything lives in the same app.